course_content/notebooks/cartopy_intro.ipynb

```diff
@@ -157,7 +157,7 @@
    "source": [
     "### Exercise 1\n",
     "\n",
-    "The following snippet of code produces coordinate arrays and some data in a rotated pole coordinate system. The coordinate system for the `x` and `y` values, which is similar to that found in the some limited area models of Europe, has a projection \"north pole\" at 177.5 longitude and 37.5 latitude."
+    "The following snippet of code produces coordinate arrays and some data in a rotated pole coordinate system. The coordinate system for the `x` and `y` values, which is similar to that found in the some limited area models of Europe, has a projection \"north pole\" at 193.0 longitude and 41.0 latitude."
```

Comment:
This isn't your fault I know, but would you mind adding the word "degrees" after each of the two numbers you've changed on this line? It will make the sentence read more nicely.